impeller/typographer/backends/skia/text_render_context_skia.cc

```diff
@@ -202,6 +203,15 @@ std::shared_ptr<GlyphAtlas> TextRenderContextSkia::CreateGlyphAtlas(
   }
 
   auto glyph_atlas = std::make_shared<GlyphAtlas>();
+  glyph_atlas->SetFontColorCallback([](const FontGlyphPair& font_glyph) {
+    SkFont sk_font(
+        TypefaceSkia::Cast(*font_glyph.font.GetTypeface()).GetSkiaTypeface(),
+        font_glyph.font.GetMetrics().point_size);
+
+    SkStrikeSpec strikeSpec = SkStrikeSpec::MakeWithNoDevice(sk_font);
+    SkBulkGlyphMetricsAndPaths paths{strikeSpec};
+    return paths.glyph(font_glyph.glyph.index)->isColor();
```

Comment:
nit: file an issue and a TODO here to use some public API from Skia for this.

also nit: maybe this belongs on impeller's `Glyph` type.

Reply:
Todo added. I agree this could probably be refactored. Lets see what we need to do for SDF and large glyphs.